<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bastfen</class>
 <widget class="QMainWindow" name="Bastfen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Bastfen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>581</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Lo√Øc</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>261</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Vie :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="loic_life">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>EFFACEUR</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Potions :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="loic_potions">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>EFFACEUR</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Arme :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="loic_weapon_name">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>EFFACEUR</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Attaque :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="loic_dmg">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>EFFACEUR</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Munitions :</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="loic_ammunitions">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>EFFACEUR</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Hugo</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>261</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Vie :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="hugo_life">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>EFFACEUR</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Potions :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="hugo_potions">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>EFFACEUR</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_15">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Arme :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="hugo_weapon_name">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>EFFACEUR</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_17">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Attaque :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="hugo_dmg">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>EFFACEUR</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_19">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Munitions :</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="hugo_ammunitions">
           <property name="font">
            <font>
             <family>Comic Sans MS</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>EFFACEUR</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>120</number>
       </property>
       <property name="topMargin">
        <number>60</number>
       </property>
       <property name="rightMargin">
        <number>120</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="pb_beat">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Lo√Øc tape Hugo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pb_drink">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Lo√Øc boit un coup</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pb_change_weapon">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Lo√Øc change d'arme</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="action_leave"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="action_leave">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
